src/wallet/rpcwallet.cpp

```diff
@@ -3287,6 +3289,14 @@ UniValue rescanblockchain(const JSONRPCRequest& request)
     return response;
 }
 
+// Deprecated account RPCs.
+UniValue getaccountaddress(const JSONRPCRequest& request) { return getlabeladdress(request); }
```

Comment:
Missing `IsDeprecatedRPCEnabled("accounts")` and throw? Same below.

Reply:
This doesn't actually use -deprecatedrpc though, that's done in #11497?

Reply:
I made the question because in the release notes says these are deprecated.